src/components/cards/FeedsCard.tsx

```diff
@@ -0,0 +1,82 @@
+import { View, Text, StyleSheet, TouchableOpacity, Image, TouchableHighlight } from "react-native";
+import React from "react";
+import { colors } from "../../constants/Colors";
+import { fonts } from "../../assets/fonts/fonts";
+import { formatDate } from "../../utils/formatTime";
+import FeedIcon from "../../assets/icons/FeedIcon";
+
+interface FeedCardProps {
+  body: string;
+  image: string | null | undefined;
+  created_at: string;
+}
+
+const FeedsCard = ({ body, image, created_at }: FeedCardProps) => {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.text}>{body}</Text>
+      <Image
+        source={{
+          uri: image,
+        }}
+        style={styles.image}
+      />
+      <View style={styles.post_details}>
+        <TouchableHighlight style={styles.flex_row}>
+          <Text style={styles.share_btn}>Share</Text>
+          {/* <FeedIcon style={styles.feed_icon} /> */}
+        </TouchableHighlight>
+        <Text style={styles.time}>{formatDate(created_at)}</Text>
+      </View>
+    </View>
+  );
+};
+
+export default FeedsCard;
+
+const styles = StyleSheet.create({
+  container: {
+    paddingHorizontal: 1,
+    borderBottomColor: colors.DROIDCONKE_LIGHT_GREY,
+    borderBottomWidth: .5,
+    padding: 25,
+  },
+  post_details: {
+    flexDirection: "row",
+    justifyContent: "space-between",
+    marginTop: 5,
+    fontSize: 16,
+    alignItems: "center",
+  },
+  text: {
+    fontFamily: fonts.MONTSERRAT_REGULAR,
+    fontSize: 16,
+    color: colors.DROIDCONKE_BLACK,
+  },
+  image: {
+    width: 318,
```

Comment:
The image is not spanning the full width of the screen
![photo_2022-11-03_14-13-28](https://user-images.githubusercontent.com/28436092/199706879-a0a519a9-4fdd-4ba4-994d-8f15cfc25fa8.jpg)

- You can set image width as a flexible property.
Change the width from `318` to  `100%` instead.